api/clamav_scanner/clamav.py

```diff
@@ -304,6 +320,28 @@ def is_clamd_running():
     return False
 
 
+def setup_clamd_daemon():
+    clamd_pid = get_clamd_pid
```

Comment:
Should this be `clamd_pid = get_clamd_pid()` or is this some Python magic I don't know about (highly likely)?

Reply:
typo. Fixed!